python/samples/concepts/chat_completion/simple_chatbot.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import asyncio
+
+from samples.concepts.chat_completion.chat_completion_services import get_chat_completion_service_and_request_settings
+from semantic_kernel.contents.chat_history import ChatHistory
+
+# This sample shows how to create a chatbot. This sample uses the following two main components:
+# - a ChatCompletionService: This component is responsible for generating responses to user messages.
+# - a ChatHistory: This component is responsible for keeping track of the chat history.
+# The chatbot in this sample is called Mosscap, who responds to user messages with long flowery prose.
+
+
+# You can select from the following chat completion services:
+# - openai
+# - azure_openai
+# - azure_ai_inference
+# - anthropic
+# - bedrock
+# - google_ai
+# - mistral_ai
+# - ollama
+# - onnx
+# - vertex_ai
+# Please make sure you have configured your environment correctly for the selected chat completion service.
+chat_completion_service, request_settings = get_chat_completion_service_and_request_settings("azure_openai")
```

Comment:
I think it would clarify if we did either a variable with the service chosen, or even an Enum of services